main.nf

```
@@ -20,7 +24,18 @@ if (!genome) error += 'No --genome provided\n'
 
 if (!annotation) error += 'No --annotation provided\n'
 
-if (merge && !metadata) {
+if (merge_mapping && merge_assembly && merge){
+  warning += 'Parameter merge ignored because --merge_mapping and --merge_assembly are supplied'
```

Good to catch this use case. However using `--merge` with these other options should throw an explicit error so the user has a chance to provide a correct command.

---

addressed in 89512c4ce883800793fb47abefe2c5d4803e3188